### ISSUE TITLE
Wtf

### ISSUE THREAD

**Issue by rugk** [jannispinter/echo720#1]
Wtf, this extension should not exist… :roll_eyes:

**Comment by jannispinter** [comment]
Yeah, don't even know how to deal with Zoom that is used to stream certain lectures :facepalm: On the other hand, I don't want to loose an entire semester, so I understand the universities who opt for existing (proprietary) tools that scale and work out of the box.

I'm trying to make the best out of it and hopefully help others. :tada:
